code/game/objects/structures/traps.dm

```diff
@@ -19,56 +33,43 @@
 		trap_effect(L)
 		animate(src, alpha = initial(alpha), time = time_between_triggers)
 
-/obj/structure/trap/examine(mob/user)
-	..()
-	if(!isliving(user)) //bad ghosts, stop trying to powergame from beyond the grave
-		return
-	user << "You reveal a trap!"
-	alpha = 200
-	animate(src, alpha = initial(alpha), time = time_between_triggers)
-
-
 /obj/structure/trap/proc/trap_effect(mob/living/L)
 	return
 
 /obj/structure/trap/stun
 	name = "shock trap"
-	desc = "A trap that will shock you, it will burn your flesh and render you immobile, You'd better avoid it."
+	desc = "A trap that will shock you and it will render you immobile. You'd better avoid it."
```

Comment:
remove "you ... it will", so it becomes "a trap that will shock and render you immobile"
(the it really doesn't work with the and)